hyperon_das_atomdb/database.py

```diff
@@ -526,12 +526,20 @@ def get_atom_as_deep_representation(self, handle: str, arity: int):
         ...  # pragma no cover
 
     @abstractmethod
-    def count_atoms(self):
+    def count_atoms(self, parameters: Optional[Dict[str, Any]] = None) -> Dict[str, int]:
         """
         Count the total number of atoms in the database.
+        If the optional parameter 'precise' is set to True returns the node count and link count (slow).
+
+        Args:
+            parameters (Optional[Dict[str, Any]]): Dict containing the following key 'precise' - sets the count
```

Comment:
This description is confuse and misleading. Confuse because it doesn't explain the difference of using or not `precise`. Misleading because I understood you changed `precise` to be a boolean but you mention string values.

Reply:
Fixed